builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -134,7 +143,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 			"automatic_restart": &schema.Schema{
 				Type:       schema.TypeBool,
 				Optional:   true,
-				Default:    true,
```

Comment:
Actually I just went into the gce api and if scheduler is not specified it will still make a scheduler with the default attributes, so keeping the default should be fine. I'm going to make the other properties of the scheduler have defaults set to avoid confusion in the future.